lib/sdl/main_sdl.cpp

```diff
@@ -1052,9 +1052,13 @@ void inputhandleText(SDL_TextInputEvent *Tevent)
 			free(utf8Buf);
 			utf8Buf = nullptr;
 		}
-		utf8Buf = UTF8toUTF32(Tevent->text, newtextsize);
+		size_t newtextsize = 0;
+		utf8Buf = UTF8toUTF32(Tevent->text, &newtextsize);
 		debug(LOG_INPUT, "Keyboard: text input \"%s\"", Tevent->text);
-		inputAddBuffer(CurrentKey, *utf8Buf);
+		for (unsigned i = 0; i < newtextsize / 4; ++i)
```

Comment:
```suggestion
		for (unsigned i = 0; i < newtextsize / sizeof(utf_32_char); ++i)
```
One tiny tweak. No issues discovered in testing thus far.